SoftLayer/fixtures/SoftLayer_Product_Package.py

```diff
@@ -892,3 +887,237 @@
         'setupFee': '0',
         'sort': 0
     }]
+
+verifyOrderDH = {
```

Comment:
Not sure what the normal process is around adding fixtures, but this seems a little long. Do we want this reduced down so it only has the fields that you need? This question is more for @allmightyspiff

Reply:
I don't really have a strong opinion on this at the moment.